### DISCUSSION TITLE
晚上的bw抢票配置可以用中午的大会员配置吗

### DISCUSSION THREAD

**Discussion by Rick-Miao** [mikumifa/biliTickerBuy#222] [提问专区]
晚上的bw抢票配置可以用中午的大会员配置吗？
![d8b0f7711af27319b74ff00c55dd1492](https://github.com/mikumifa/biliTickerBuy/assets/40517817/a9d7ffa9-4a71-4a35-8390-7e8247630a5d)

**Comment by MegumiN152** [comment]
晚上选项会变吧
